index.html

```diff
@@ -652,26 +652,34 @@ <h3>
         </h3>
         <p>
           To <dfn data-lt="prompt the user to choose|prompting the user to choose" class=
-          "export">prompt the user to choose</dfn> one of several |options| associated with a
-          |descriptor|, the user agent must perform the following steps. This algorithm returns
-          either {{PermissionState/"denied"}} or one of the options.
+          "export">prompt the user to choose</dfn> one or more |options| associated with a given
+          |descriptor:PermissionDescriptor| and an optional <a>boolean</a> |allowMultiple:boolean|
+          (default false), the user agent must perform the following steps. This algorithm returns
+          either {{PermissionState/"denied"}} or the user's selection.
         </p>
         <ol class="algorithm">
           <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"denied"}}, return
           {{PermissionState/"denied"}} and abort these steps.
           </li>
           <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"granted"}}, the user
-          agent may return one of |options| and abort these steps. If the user agent returns
-          without prompting, then subsequent <a data-lt="prompt the user to choose">prompts for the
-          user to choose</a> from the same set of options with the same |descriptor| must return
-          the same option, unless the user agent receives <a>new information about the user's
-          intent</a>.
+          agent may return one or more of |options| chosen by the user and abort these steps. If the user agent
```

Comment:
```suggestion
          agent may return one (or more if |allowMultiple| is true) of |options| chosen by the user and abort these steps. If the user agent
```

To incorporate `allowMultiple` into the behavior.